config/services.yaml

```diff
@@ -4,7 +4,8 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
-
+    mailer_from: '%env(resolve:MAILER_FROM_ADDRESS)%'
```

Comment:
You probably should modify the installation doc (or at least the README) to tell that a new environnement variable is needed